ADOPTERS.md

```diff
@@ -6,3 +6,4 @@ This really helps the project to gain momentum and credibility. It's a small con
 | Organization | Contact | Description of Use |
 |--------------|---------|--------------------|
 | Sovereign Cloud Stack | https://scs.community/ | Currently in the process of evaluating how to generate a complete SBOM of our stack and secure the supply chain. We plan to use ORT to analyze our reference implementation as well as to add ORT to our [recommended operator's toolset](https://scs.community/about/#technological-vision). |
+| Deutsche Telekom AG | https://www.telekom.de/ | Deutsche Telekom is in the process of adopting ORT as part of our CI/CD chain and is working on a custom ORT reporter in our ORT fork at https://github.com/telekom/ort. | 
```

Comment:
@daniel-eder  For contact I would prefer an email address or a GitHub username (daniel-eder) - we should fix this for SCS as well

Reply:
Ping @itrich. Maybe also have an extra URL column then?

Reply:
For now I replaced the URL with an e-mail address (as I don't think there's a good way to establish contact via github username).

Reply:
Done for the DT entry

Reply:
@daniel-eder  Are you Ok with me squashing your 3 commits into a single one?

Reply:
@tsteenbe of course!